src/olympia/bandwagon/models.py

```diff
@@ -117,8 +117,6 @@ class Collection(ModelBase):
 
     objects = CollectionManager()
 
-    top_tags = TopTags()
```

Comment:
The `TopTags` at the beginning of the file is now unused and can be removed as well.